test/cases/coerced_tests.rb

```diff
@@ -1483,3 +1483,15 @@ def test_has_primary_key_coerced
     original_test_has_primary_key
   end
 end
+
+
+
+
+module ActiveRecord
+  module ConnectionAdapters
+    class ReaperTest < ActiveRecord::TestCase
+      # Coerce can be removed if Rails version > 6.0.3
+      coerce_tests! :test_connection_pool_starts_reaper_in_fork
```

Comment:
You could coerce only if `Process` does not respond to `:fork` as Rails did originally.

Reply:
Done